File: rules-interpreter/src/main/java/com/github/iamrenny/ruleflow/visitors/GrammarVisitor.java
package com.github.iamrenny.ruleflow.visitors;

import com.github.iamrenny.ruleflow.RuleFlowLanguageBaseVisitor;
import com.github.iamrenny.ruleflow.RuleFlowLanguageParser;

public class GrammarVisitor extends RuleFlowLanguageBaseVisitor<String> {

    @Override
    public String visitParse(RuleFlowLanguageParser.ParseContext ctx) {
        return removeSingleQuote(ctx.workflow().workflow_name().getText());
    }

    private String removeSingleQuote(String text) {
        return text.replace("'", "");
    }
}

File: analang-interpreter/src/main/kotlin/com/rappi/fraud/rules/parser/visitors/GrammarVisitor.kt
package com.rappi.fraud.rules.parser.visitors

import com.rappi.fraud.analang.ANABaseVisitor
import com.rappi.fraud.analang.ANAParser
import com.rappi.fraud.rules.parser.removeSingleQuote

class GrammarVisitor : ANABaseVisitor<String>() {

    override fun visitParse(ctx: ANAParser.ParseContext): String {
        return ctx.workflow().workflow_name().text.removeSingleQuote()
    }
}